### DISCUSSION TITLE
Manifest not found for component id "96dcffa8-2c69-47ec-ba5c-770213aeb4fe" and version "1.0.0". (All web parts manifests not found) + App not in 'my apps'

### DISCUSSION THREAD

**Discussion by Forest-runklebc** [microsoft-search/pnp-modern-search#1529] [Q&A] [answered]
I am attempting to install the v4 sppkg and after installing I am continually receiving errors when loading the web parts. Every single web part gives a different manifest error after installing the sppkg. The same result happens when trying to use the previous v3 sppkg.

Reproduce:

1) Download v4 sppkg
2) Drag and drop onto the "Apps for Sharepoint" page
3) Click the checkbox to allow for use on all sites
4) Edit sharepoint page to add web part
5) Select any web part from the pnp-search
6) Receive error

Web Part Error:
![image](https://user-images.githubusercontent.com/5815376/141853212-51bb7ba0-886b-4df1-8983-4d0610f89e2e.png)

SPPKG Installed Successfully:
![image](https://user-images.githubusercontent.com/5815376/141853378-995354c7-7948-43a0-b4d0-4a4eb3ff5c84.png)

Available Apps to Add From My Organization Empty:
![image](https://user-images.githubusercontent.com/5815376/141853446-8d290802-92c9-415a-8dc9-d42798a07fd4.png)

Full text error:

Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
[SPLoaderError.loadComponentError]:
Failed to load component "544c1372-42df-47c3-94d6-017428cd2baf" (pnpSearchResultsWebPart). Original error: Manifest not found for component id "96dcffa8-2c69-47ec-ba5c-770213aeb4fe" and version "1.0.0".

INNERERROR:
Manifest not found for component id "96dcffa8-2c69-47ec-ba5c-770213aeb4fe" and version "1.0.0".
CALLSTACK:
Error
    at t [as constructor] (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:67:19894)
    at new t (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:9374)
    at Function.e.buildErrorWithVerboseLog (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:46574)
    at Function.e.buildLoadComponentError (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:42619)
    at https://modernb.akamai.odsp.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_0e72155ab8ba4215d52d.js:1:11895

---
Filter text error:

Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
[SPLoaderError.loadComponentError]:
Failed to load component "544c1372-fb1d-4e96-bc1e-31fd66979667" (pnpSearchFiltersWebPart). Original error: Manifest not found for component id "96dcffa8-2c69-47ec-ba5c-770213aeb4fe" and version "1.0.0".

INNERERROR:
Manifest not found for component id "96dcffa8-2c69-47ec-ba5c-770213aeb4fe" and version "1.0.0".
CALLSTACK:
Error
    at t [as constructor] (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:67:19894)
    at new t (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:9374)
    at Function.e.buildErrorWithVerboseLog (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:46574)
    at Function.e.buildLoadComponentError (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_2548dff9353f4dfa701500989e0eb887.js:68:42619)
    at https://modernb.akamai.odsp.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_0e72155ab8ba4215d52d.js:1:11895

**Comment by Forest-runklebc** [comment]
@wobba - Are there site settings that must be enabled in sharepoint to allow these packages to run successfully? I have been browsing past issues with 'manifest not found' and I have seen the solution was typically to remove and reinstall and enable tenant wide deployment. Additionally some past solutions were to install the search-extensibility package which I see has been moved to one singular .sppkg as of v4. I have tried removing and re-adding numerous times and continue to get this error.

**Comment by Forest-runklebc** [comment, marked as answer]
Solution found: 

Manifests directly out of the downloaded .sppkg from the repository were not correct for some reason. After downloading the repository, opening it in an IDE, and running in the IDE embedded console:

gulp clean
gulp build
gulp bundle --ship
gulp package-solution --ship

And deploying the .sppkg it generates, the webparts loaded successfully and are functioning as expected. Note anyone in the future with this problem you will have to install all required packages to successfully build the solution as the errors pop up one by one with npm install [package-here] until no errors are thrown and it builds successfully.

**Comment by wobba** [comment]
Haven't had any issues with the sppkg's built so far. And seems your screenshot was v4.0. while the current is v4.4.
